examples/test_tasks.yaml

```diff
@@ -62,10 +62,10 @@ spec:
         volumes:
             - name: gac
               hostPath:
-                path: ${workspace}/config/gac.json
+                path: /workspace/config/gac.json
             - name: cloudconfig
               hostPath:
-                path: ${workspace}/config/gcloud
+                path: /workspace/config/gcloud
```

Comment:
the path here refers to `/workspace` while the volume mount  int the step refers to `/root`

Reply:
hm I think that's okay, the docs for `hostPath` (here) say:

> HostPath represents a pre-existing file or directory on the host machine that is directly exposed to the container. 

So I think what this is doing is creating a volume from a dir that exists already on the host (which was mounted into `/workspace` via the resource), then in volume mounts it's mounting this dir into a specific location in `/root`